src/components/toolbar/ToolBar.vue

```diff
@@ -166,4 +188,4 @@ export default {
   },
 };
 </script>
-<style lang="scss" src="./toolbar.scss" scoped />
+<style lang="scss"  src="./toolbar.scss" />
```

Comment:
Remove the space added:
```suggestion
<style lang="scss" src="./toolbar.scss" />
```